kmk/scanners/native_keypad_scanner.py

```diff
@@ -1,6 +1,6 @@
 import keypad
 
-from kmk.matrix import DiodeOrientation
+from kmk.matrix import DiodeOrientation, KeyEvent
```

Comment:
`kmk.matrix.KeyEvent` is a backwards-compatibility implementation of the CP `keypad.KeyEvent`.
Please don't use the compatibility version instead of the proper one. See below.